mbedtls/vita.patch

```diff
@@ -0,0 +1,143 @@
+diff --git a/library/entropy_poll.c b/library/entropy_poll.c
+index b5024c83f..d42495325 100644
+--- a/library/entropy_poll.c
++++ b/library/entropy_poll.c
+@@ -41,7 +41,7 @@
+ 
+ #if !defined(unix) && !defined(__unix__) && !defined(__unix) && \
+     !defined(__APPLE__) && !defined(_WIN32) && !defined(__QNXNTO__) && \
+-    !defined(__HAIKU__) && !defined(__midipix__)
++    !defined(__HAIKU__) && !defined(__midipix__) && !defined(__vita__)
+ #error \
+     "Platform entropy sources only work on Unix and Windows, see MBEDTLS_NO_PLATFORM_ENTROPY in mbedtls_config.h"
+ #endif
+@@ -153,6 +153,27 @@ static int sysctl_arnd_wrapper(unsigned char *buf, size_t buflen)
+ #endif /* KERN_ARND */
+ #endif /* __FreeBSD__ || __NetBSD__ */
+ 
++/*
++ * vitasdk provides getentropy(), with 256b max per call.
++ */
++#if defined(__vita__)
++int getentropy(void *, size_t); // hidden behind __BSD_VISIBLE
```

Comment:
@scribam can you make an issue on vitasdk/newlib about making the `sys/unistd.h` header?